src/android.ts

```diff
@@ -839,6 +923,36 @@ export const androidApi = <RunTarget extends SupportedRunTarget<'android'>>(
         )
             throw new Error('Failed to set proxy.');
     },
+    addCalendarEvent: async (eventData) => {
+        await adb(
+            [
+                'shell',
+                'am',
+                `start -a android.intent.action.INSERT -t 'vnd.android.cursor.dir/event' --el beginTime '${eventData.startDate.valueOf()}' --es title '${
+                    eventData.title
+                }' --el endTime '${eventData.endDate.valueOf()}'`,
+            ],
+            { reject: true }
+        );
+
+        await pause(3000); // wait for the calendar app to open
+        await adb(['shell', 'input', 'keyevent', '3']); // Home button, the apps is closed and creates the event
+    },
+    addContact: async (contactData) => {
+        if (!options.capabilities.includes('frida'))
+            throw new Error('Frida is required to add contacts to the contact book.');
+
+        const device = await frida.getUsbDevice();
+        const pid = await device.spawn('com.android.contacts');
```

Comment:
The docstring needs to mention that this only works if `com.android.contacts` is installed.

On my moto that I've kept stock (i.e. not installed Lineage on), that app isn't installed and trying to spawn it fails. It only has `com.google.android.contacts`:

```sh
❯ adb shell cmd package list packages | grep contact
package:com.google.android.contacts
package:com.motorola.contacts.preloadcontacts
package:com.google.android.syncadapters.contacts
package:com.android.providers.contacts
```

Reply:
Ah, yes. I should work with that, too, so maybe we should check and start `com.google.android.contacts` instead if that exists?

Reply:
I don't have a rooted device with Google apps handy, so we'll postpone this. Moved to #135.